<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionZoneDialog</class>
 <widget class="QDialog" name="DetectionZoneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>198</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>198</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>198</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detection Zone Configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_PixelsPerMeter"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Direction"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_AcceptanceAngle"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_PixelsPerMeter">
      <property name="text">
       <string>Pixels Per Meter</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Direction">
      <property name="text">
       <string>Direction (Cardinal Deg.)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_AcceptanceAngle">
      <property name="text">
       <string>Acceptance Angle</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox_Directional">
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Directional">
      <property name="text">
       <string>Directional</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_ZoneName">
      <property name="text">
       <string>Zone Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_ZoneName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_ZoneName</tabstop>
  <tabstop>lineEdit_PixelsPerMeter</tabstop>
  <tabstop>checkBox_Directional</tabstop>
  <tabstop>lineEdit_Direction</tabstop>
  <tabstop>lineEdit_AcceptanceAngle</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DetectionZoneDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DetectionZoneDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Directional</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_Direction</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Directional</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_AcceptanceAngle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
